superset/translations/zh/LC_MESSAGES/messages.json

```diff
@@ -196,38 +222,52 @@
-      "Request is not JSON": [""],
-      "Request is incorrect: %(error)s": [""],
-      "`confidence_interval` must be between 0 and 1 (exclusive)": [""],
+      "Name must be unique": ["名称必须是唯一的"],
+      "Deleted %(num)d chart": ["删除了 %(num)d 个图表"],
+      "Request is not JSON": ["请求不是JSON"],
+      "Request is incorrect: %(error)s": ["请求不正确: %(error)s"],
+      "`confidence_interval` must be between 0 and 1 (exclusive)": [
+        "`置信区间` 必须介于0和1之间（独占）"
+      ],
       "lower percentile must be greater than 0 and less than 100. Must be lower than upper percentile.": [
-        ""
+        "下百分位数必须大于0且小于100。而且必须低于上百分位"
       ],
       "upper percentile must be greater than 0 and less than 100. Must be higher than lower percentile.": [
-        ""
+        "上百分位数必须大于0且小于100。而且必须高于下百分位。"
+      ],
+      "`width` must be greater or equal to 0": ["`宽度` 必须大于或等于0"],
+      "`row_limit` must be greater than or equal to 1": [
+        "`行限制` 必须大于或等于1"
+      ],
+      "`row_offset` must be greater than or equal to 0": [
+        "`行偏移量` 必须大于或等于0"
+      ],
+      "There are associated alerts or reports: %s,": [
+        "存在关联的警报或报告：%s,"
       ],
-      "`width` must be greater or equal to 0": [""],
-      "`row_limit` must be greater than or equal to 1": [""],
-      "`row_offset` must be greater than or equal to 0": [""],
-      "There are associated alerts or reports: %s,": [""],
-      "Database does not exist": [""],
+      "Database does not exist": ["数据库不存在"],
       "Dashboards do not exist": ["看板"],
-      "Datasource type is required when datasource_id is given": [""],
-      "Chart parameters are invalid.": [""],
-      "Chart could not be created.": ["您的查询无法保存"],
-      "Chart could not be updated.": ["您的查询无法保存"],
-      "Chart could not be deleted.": ["这个查询无法被加载"],
-      "There are associated alerts or reports": [""],
-      "Changing this chart is forbidden": [""],
+      "Datasource type is required when datasource_id is given": [
+        "给定数据源id时，需要提供数据源类型"
+      ],
+      "Chart parameters are invalid.": ["图表参数无效。"],
+      "Chart could not be created.": ["您的查询无法保存。"],
+      "Chart could not be updated.": ["您的查询无法保存。"],
+      "Chart could not be deleted.": ["这个查询无法被加载。"],
+      "There are associated alerts or reports": ["存在关联的警报或报告"],
+      "Changing this chart is forbidden": ["禁止更改此图表"],
       "Charts could not be deleted.": ["这个查询无法被加载"],
-      "Import chart failed for an unknown reason": [""],
-      "Owners are invalid": [""],
+      "Import chart failed for an unknown reason": ["导入图表失败，原因未知"],
+      "Owners are invalid": ["所有者无效"],
       "Dataset does not exist": ["数据集不存在"],
-      "`operation` property of post processing object undefined": [""],
-      "Unsupported post processing operation: %(operation)s": [""],
-      "Adding new datasource [{}]": ["添加 Druid 数据源"],
-      "Refreshing datasource [{}]": [""],
+      "`operation` property of post processing object undefined": [
+        "处理对象的 `操作` 属性未定义"
```

Comment:
@ktmud this line isn't very clear, can you help?

Reply:
```
后处理未指定操作类型（`operation`）
```